build_debian.sh

```diff
@@ -92,7 +100,11 @@ trap_push 'sudo umount $FILESYSTEM_ROOT/proc || true'
 sudo LANG=C chroot $FILESYSTEM_ROOT mount proc /proc -t proc
 
 ## Pointing apt to public apt mirrors and getting latest packages, needed for latest security updates
+if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
+sudo cp files/apt/sources.list$ARCH_SUFFIX $FILESYSTEM_ROOT/etc/apt/sources.list
```

Comment:
Add indentation in if-else blocks?

Reply:
Done, by maintaining same source.list with [arch=] option